### DISCUSSION TITLE
Naming suggestion

### DISCUSSION THREAD

**Discussion by skhaz** [janbjorge/pgqueuer#85] [Ideas]
Great project, I am a heavy user of task queues.

If I may, I would like to recommend you to use lowercase on package name to be in conformity with other libraries.

Thank you.

**Comment by janbjorge** [comment]
I agree that transitioning to a lowercase package name would align better with standard Python practices. I'm currently exploring ways to implement this change without disrupting existing users. Any suggestions are highly appreciated.

**Comment by skhaz** [comment]
I think you can create a symbolic link, or manually duplicate it.

**Comment by janbjorge** [comment]
Completed by: https://github.com/janbjorge/pgqueuer/pull/89
